src/features/dotnetTest.ts

```diff
@@ -511,7 +471,7 @@ class DebugEventListener {
                     if (!isStarted) {
                         reject(err.message);
                     } else {
-                        this._outputChannel.appendLine("Warning: Communications error on debugger event channel. " + err.message);
+                        this._eventStream.post(new DebuggerWarning(`Communications error on debugger event channel. ${err.message}`));
```

Comment:
Let's rename this event to mention that it part of the DotNetTest feature and not the general debugger.

Reply:
@DustinCampbell I resolved all the names as per your suggestion. Please take a look